### ISSUE TITLE
Sketch out possible UX's

### ISSUE THREAD

**Issue by misterpeddy** [misterpeddy/hands-down#3]
A few things that come to mind:

* What will the page look like? 
* Are there any interactions? 
* How to educate the users about what's happening?
* What is the notification mechanism (what's version 0 and what's version 10)?

**Comment by Berkmann18** [comment]
> What will the page look like?
I haven't got round making mood boards/mockups yet so not sure about the page look.

> Are there any interactions?
I guess so, the interactions would be the (assuming the collection settings wouldn't be present on the user side):
- Camera view of the user (like to sketching the possibleon your Proof of Concept).
- Notifications (say when there's a contact detected by the model and perhaps when the number of hand/face contact exceeds a threshold).
- _maybe something to do with friends/other people, like: X touched her face n times, can you do better?_

> How to educate the users about what's happening?
1. Inform them about data being collected (to comply with GDPR/... and to ensure them that they aren't being spied on)
2. Potentially having a (basic) walkthrough of what the web app is doing (so users can see what is happening when they allow the web app to use their webcam).

> What is the notification mechanism (what's version 0 and what's version 10)?

Not sure about the version thingy, but the notification mechanism could be like the following:

![HandsDown](https://user-images.githubusercontent.com/8260834/77675326-b6b67b00-6f84-11ea-85a5-33b3a27d24f3.png)
Basically a system triggered by the hand/face contact detection.
